domain/optatask/src/main/resources/org/optatask/solver/taskAssigningSolverConfig.xml

```diff
@@ -26,7 +26,8 @@
   <daemon>true</daemon>
 
   <scoreDirectorFactory>
-    <ksessionName>optataskKsession</ksessionName>
+    <!--<ksessionName>optataskKsession</ksessionName>-->
```

Comment:
@ge0ffrey Setting this property is necessary to deploy the module in a KIE server container, however it results in an exception while building the scoreDirectorFactory. Uncommenting it and running the realTimeTest throws java.lang.IllegalArgumentException: If ksessionName (optataskKsession) is not null, then the kieContainer (null) must not be null.